### PR TITLE
Add pre-requisite and description on patternfiles

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: samples/IstioFilterPattern.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
+#Pre-requisite: A Kubernetes cluster  with a running Istio control plane
+#Description: This Patternfile injects rate limiter filter in the dataplane in istio-test namespace
```
Comment (leecalcote) [inline samples/IstioFilterPattern.yaml:2]:
```suggestion
#Description: This design injects the rate limiter filter into the dataplane of the `default` namespace
```

Wording change (no more "patternfile") and filters should not be injected into the Istio control plane, but into workload-bearing namespaces.

File: samples/IstioSPIRE.yaml [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#Pre-requisite: A Kubernetes cluster 
+#Description: This Patternfile installs Spire with the default configuration
```
Comment (leecalcote) [inline samples/IstioSPIRE.yaml:2]:
```suggestion
#Description: This design deploys a SPIRE-configured Istio service mesh with the default SPIRE configuration.
```

File: samples/bookInfoPattern.yaml [hunk 3]
```diff
@@ -1,4 +1,6 @@
-name: MesheryGeneratedPatternFile
+#Pre-requisite: A running Istio control plane in a Kubernetes cluster
+#Description: This Patternfile deploys Istio's demo Book info application in default namespace
```
Comment (leecalcote) [inline samples/bookInfoPattern.yaml:2]:
```suggestion
#Description: This design deploys the "Book Info" sample application in the `default` namespace.
```

File: samples/bookstore.yaml [hunk 4]
```diff
@@ -1,4 +1,6 @@
-name: Autogenerated
+#Pre-requisite: A Kubernetes cluster 
+#Description: This Patternfile deploys OSM's demo Book store application in bookstore namespace
```
Comment (leecalcote) [inline samples/bookstore.yaml:2]:
```suggestion
#Description: This design deploys the Book Store sample application into the `bookstore` namespace.
```

File: samples/emojivoto.yaml [hunk 5]
```diff
@@ -1,3 +1,5 @@
+#Pre-requisite: A Kubernetes cluster 
+#Description: This Patternfile deploys Linkerd's demo Emojivoto application in emojivoto namespace
```
Comment (leecalcote) [inline samples/emojivoto.yaml:2]:
```suggestion
#Description: This design deploys Linkerd's "Emojivoto" sample application in `emojivoto` namespace. Sample applications from one service mesh project can be deployed to different service mesh.
```

File: samples/istioinstall.yaml [hunk 6]
```diff
@@ -1,3 +1,5 @@
+#Pre-requisite: A Kubernetes cluster 
+#Description: This Patternfile installs Istio version 1.12.0 control plane in the istio-system namespace with default profile.
```
Comment (leecalcote) [inline samples/istioinstall.yaml:2]:
```suggestion
#Pre-requisite: A Kubernetes cluster.
#Description: This design installs the Istio service mesh version 1.12.0 control plane in the `istio-system` namespace with default profile.
```

File: samples/IstioSPIRE.yaml (hunk 2 above)
Comment (Revolyssup) [reply samples/IstioSPIRE.yaml:2]: Actually it does not deploy Istio service mesh. It just deploys Spire server and agent configured and ready for an Istio deployment.

File: samples/bookstore.yaml (hunk 4 above)
Comment (Revolyssup) [reply samples/bookstore.yaml:2]: Yes

File: samples/minimalistiobookinfo.yaml [hunk 7]
```diff
@@ -1,3 +1,5 @@
+#Pre-requisite: A Kubernetes cluster 
+#Description: This Patternfile installs Istio control plane version 1.12.0 with minimal configuration and then installs Istio's book info application.
```
Comment (leecalcote) [inline samples/minimalistiobookinfo.yaml:2]:
```suggestion
#Pre-requisite: A Kubernetes cluster. 
#Description: This design installs the Istio control plane version 1.12.0 with minimal configuration and deploys the "Book Info" sample application. 
```

**Review by leecalcote — DISMISSED** [review]

File: samples/IstioSPIRE.yaml (hunk 2 above)
Comment (leecalcote) [reply samples/IstioSPIRE.yaml]: Oh, I see.

**Pull request merged**